### ISSUE TITLE
Is there a more up to date alternate repo?

### ISSUE THREAD

**Issue by nonplayer** [ceryliae/DnDAppFiles#308]
Seems this one hasn't seen any commits since June, and from the Dropbox note the owner has ceased working on the project. Is anyone maintaining a more up to date and regularly updated fork?

**Comment by NeilMartin** [comment]
I've been toying with the idea of forking this repo so I could change the format of the files slightly.  I'm also interested in any other repos that are still maintained or may have done this already.

**Comment by christiantillich** [comment]
I'm getting permissions  errors when trying to publish a new branch, but I'd be happy to contribute if someone could make me a contributor?

**Comment by kabikuchi** [comment]
Not sure if this repo is active, there are several PR's pending. There are brand new spells from XGtE and I am working on updating classes. I would be happy to have active repo (fork?) where people could contribute.

**Comment by ceryliae** [comment]
Hey guys, I’m not actively working on the repo but I will look at these
changes and and merge the pull requests later today.

**Comment by kabikuchi** [comment]
Thanks! That's great :)

**Comment by bemyak** [comment]
I was looking for any repo with up-to-date D&D resouces and I found `astranauta/astranauta.github.io`
It uses JSON to store data and gets regular updates.

**Comment by kabikuchi** [comment]
It's nice, but on first glance it doesn't seem to be compatible with lion den's app.

**Comment by bemyak** [comment]
Yes, of course it doesn't. But some people uses this repo for their own projects as source of data.

**Comment by kabikuchi** [comment]
Uhm, it was just taken down because of copyright violation...

**Comment by storskegg** [comment]
I have a relatively up-to-date fork. I've merged a few PRs into it, too. @ceryliae would you like me to issue a PR so you can compare it, and the network, against the current outstanding PRs into your repo? It may save you a little time.

**Comment by jplcny** [comment]
Liam,

I’m working on the ToA monsters, and should have them done in the next few days. Are you inviting others to your repo?

**Comment by storskegg** [comment]
@jplcny I'll wait to see what @ceryliae wants/has time to do.

**Comment by jplcny** [comment]
I was under the impression that ceryliae backed off of this project. Am I mistake ?

**Comment by storskegg** [comment]
>Hey guys, I’m not actively working on the repo but...

Oh, you're right. So, since @ceryliae mentioned looking at the PRs, I'll let that settle first. That also gives me a chance to get caught up with some other development I need to have done by the end of the month.

After that, if @ceryliae is interested in retiring from the core repo, here, I'd be happy to take up the mantle. Re: the Dropbox comments, above, that's something I'd be willing to cover, as well. I already maintain a copy of the repo in my Dropbox for my own personal use.

**Comment by ceryliae** [comment]
Hey guys, I didn’t get a chance to look at the PRs last night. I’ll do my
best to do it today. Liam, I would be interested in passing the reins over
to you. Could we talk on skype or discord first?

**Comment by storskegg** [comment]
@ceryliae Absolutely. Feel free to ping me on discord: storskegg#2969

**Comment by jplcny** [comment]
Sounds excellent. And if you need a backup man to delegate stuff to, I’m available.

**Comment by NeilMartin** [comment]
Oh no, did anyone download/back up astranauta/astranauta.github.io ? I looked at it yesterday and it looked promising, but now it's gone.

**Comment by ceryliae** [comment]
Liam, I sent you a friend request on discord

**Comment by nonplayer** [comment]
I'm also more than willing to contribute to maintaining this and/or any Dropbox-related tasks

Discord: nonplayer#0435

**Comment by morgdalaine** [comment]
There is someone on Twitter @FightClubXML who has done a nice job all by
himself it seems. Dropbox link: https://t.co/nPWpPX7Wgq



- maddi

**Comment by nonplayer** [comment]
Just checking in to confirm, is there a better fork of this to follow, and/or a beter dropbox repo to subscribe to? I want to get the Xanathar's updates for my current games.

**Comment by morgdalaine** [comment]
twitter dude @fightclubxml is what i've switched to.

**Comment by LittIeFreaky** [comment]
Is this repository dead or did anyone keep updating it? I'd join in updating.

**Comment by jplcny** [comment]
I think it might dead, a victim of D&D Beyond. If someone can get ownership
of the repo, I’d be willing to lend a hand.

**Comment by LittIeFreaky** [comment]
@jplcny is D&D Beyond better? Didn't try it yet, been looking for an app that is easy to use, if possible on different OSs. Does D&D Beyond support homebrew additions as well as all the expansions in here?

**Comment by jplcny** [comment]
It does, but you have to pay for them. You can enter your own home brew
monsters, spells, races, subclasses (but not base classes). It’s a web app,
so you can use it from any device with a web browser and an internet
connection. Also, it doesn’t have a combat manager yet.

**Comment by zward88** [comment]
I've been using Improved Initiative for combat management.  That's what I've been using these files for.  I've also been working on creating the content for Mordenkainen's Tome of Foes for use in that app.  I've been expanding on the project ceryliae started, but I think the repo is dead.

**Comment by jplcny** [comment]
Perhaps those of us who are still actively using the files can start up a
new repo. I'm willing to contribute to the creation of the xml files.

**Comment by LogosWorks** [comment]
Take a look to https://discord.gg/nGvRCDs There are resources here.

**Comment by zward88** [comment]
I'm working on converting what I have now, as it's in a different format.  Custom JSON for a web app that I use.  I'm working on the conversion script now...lots of regex.  I don't want to lose the work I've already put into it.

**Comment by lkhedlund** [comment]
This is the most up-to-date fork I've found: [https://github.com/Cphrampus/DnDAppFiles](https://github.com/Cphrampus/DnDAppFiles)
